test/performance/search/search_benchmark.cpp

```diff
@@ -0,0 +1,334 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/search/algorithm/all.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+
+using namespace seqan3;
+using namespace seqan3::test;
+
+struct options
+{
+    size_t const sequence_length;
+    bool const has_repeats;
+    size_t const number_of_reads;
+    size_t const read_length;
+    double const prob_insertion;
+    double const prob_deletion;
+    uint8_t const simulated_errors;
+    uint8_t const searched_errors;
+    uint8_t const strata;
+    double const standard_deviation = 0;
```

Comment:
```suggestion
    double const standard_deviation{0};
```

I'd call it `stddev` since it's a quite common abbreviation.